### PR TITLE
Fix/bundle genesis data

### PR DESCRIPTION
**Pull request by zone117x**
Fixes https://github.com/blockstack/stacks-blockchain-api/issues/393

The chainstate.txt is now committed to the repo, and the locked STX data from v1 is automatically imported when ran in production mode.

Optimized the import process for token vesting. Previously took around 20-30 seconds on my machine, now takes less than 2 seconds.

The committed chainstate.txt is gzipped to only ~8MB. Uses NodeJS's native zlib to decompress, with essentially zero overhead.

### CODE REVIEW

File: src/import-v1/c32helper.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// The b58ToC32 function from https://github.com/blockstack/c32check, optimized for speed in NodeJS.
```
Comment (asimm241) [inline src/import-v1/c32helper.ts:1]: is there a need for separate test cases for this?

**Review by asimm241 — APPROVED** [review]
other than my comment above , LGTM.

File: src/import-v1/c32helper.ts (hunk 1 above)
Comment (zone117x) [reply src/import-v1/c32helper.ts:1]: Tests added

**Review by asimm241 — APPROVED** [review]
@zone117x LGTM!

**Review by noumanahmads — APPROVED** [review]
@zone117x Looks good to me.

**Pull request merged**